kpi/deployment_backends/kc_access/shadow_models.py

```diff
@@ -250,7 +255,7 @@ class KobocatUserObjectPermission(ShadowModel):
     UserObjectPermissionManager, is NOT included!
     """
     permission = models.ForeignKey(KobocatPermission, on_delete=models.CASCADE)
-    content_type = models.ForeignKey(KobocatContentType, on_delete=models.CASCADE)
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
```

Comment:
Yes, with Kc 2.0, `KobocatContentType` started throwing a lot of angry errors because the ContentType model in KC and KPI should now match because they both are on Django 2.2

Reply:
Hmmm. Let see this together.

Reply:
I've reverted `ContentType` to `KobocatContentType` for now, however, I am still receiving this error `
```
kpi_1             | user_statistics.KobocatUserObjectPermission.content_object: (contenttypes.E004) 'KobocatUserObjectPermission.content_type' is not a ForeignKey to 'contenttypes.ContentType'.
kpi_1             | 	HINT: GenericForeignKeys must use a ForeignKey to 'contenttypes.ContentType' as the 'content_type' field.
```